docs/02.API-REFERENCE.md

```diff
@@ -11872,6 +11917,90 @@ main (void)
 - [jerry_generate_snapshot](#jerry_generate_snapshot)
 - [jerry_exec_snapshot](#jerry_exec_snapshot)
 
+## jerry_get_source_info
+
+**Summary**
+
+Returns a newly created source info structure corresponding to the passed script/module/function.
+The function is lower level than `toString()` operation, but provides more contextual information.
+
+*Notes*:
+- Returned value must be freed with [jerry_free_source_info](#jerry_free_source_info) when it
+is no longer needed.
```

Comment:
It should be mentioned that this method requires the `JERRY_FUNCTION_TO_STRING` build option to be enabled otherwise it will only return NULL values.